client/source/components/questionsAnswers/AnswerList.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import AnswerItem from './AnswerItem';
+import UnorderedList from './styles/UnorderedList';
+import { MoreButtonContainer } from './styles/Containers.styled';
+
+const AnswerList = function CreateAnswerListComponent({ answers, getQuestions }) {
+  const [count, setCount] = useState(2);
+  const answerKeys = Object.keys(answers);
+  const currentAnswerKeys = answerKeys.slice(0, count);
```

Comment:
No action, just a comment. I like how you have this up here - keeps the rendering below clean. Gives me some inspiration to refactor.

Reply:
Awesome!